### ISSUE TITLE
Are there still windows 7 64 exe, being produced?

### ISSUE THREAD

**Issue by Dboingue** [DanielUranga/trainingdata-tool#15]
The links within the closed issue are no valid anymore. I think that the XML error page I end up with means that.
https://github.com/DanielUranga/trainingdata-tool/issues/3#issue-415070260

If however, there are no updated such executable, then I figure that this repository can be used in visual studio, and will compile fine on that plaform? right.  I am new to this, (github repository structures, and anything new from the past 10 years in terms of visual studio, but i am making some progress), and using an executable would spare me the delay on compiling, etc...

**Comment by Checkmate6659** [comment]
i also want to build this tool because i didn't find an executable, and so i tried opening the solution in visual studio. but it says it can't find the .vxcproj file, and so i cant build it.

**Comment by KyleGass** [comment]
---------------------------------------------
Will be updated soon
-----------------------
This Link by the LC0 discord server seems to work:
https://cdn.discordapp.com/attachments/427066771627966466/787663547546861568/trainingdata-tool.exe

For anyone following this LC0 guide: Training and Testing a Net on Google Colab: Beginner Friendly Guide
(https://lczero.org/dev/wiki/training-and-testing-a-net-on-google-colab-beginner-friendly-guide/)

I later also needed to create a proto folder in lczero-training-master\tf and put in this file:
https://github.com/LeelaChessZero/lczero-common/tree/master/proto

Discord fildes maybe too old (https://cdn.discordapp.com/attachments/427066771627966466/822101986597797918/chunk_pb2.py
https://cdn.discordapp.com/attachments/427066771627966466/822101992587395142/net_pb2.py)
(and also created a tmp folder with an empty mymodel4.txt file in that folder)

I also had to edit the example.yaml file in the lczero-training-master\tf\configs folder

%YAML 1.2
---
name: t1-128x10b
gpu: 0
dataset:
  num_chunks: 16865
  train_ratio: 0.95
  input_train: /content/gdrive/My Drive/lc0/Training Data/*/
  input_test: /content/gdrive/My Drive/lc0/Training Data/*/
training:
  batch_size: 512
  test_steps: 2000
  train_avg_report_steps: 1
  total_steps: 100000
  checkpoint_steps: 1000
  warmup_steps: 250
  shuffle_size: 524288
  lr_values:
    - 0.02
    - 0.002
    - 0.0005
  lr_boundaries:
    - 60000
    - 80000
  policy_loss_weight: 1
  value_loss_weight: 1
  moves_left_loss_weight: .01
  path: /content/gdrive/My Drive/lc0/
model:
  filters: 128
  residual_blocks: 10
  se_ratio: 8
...

I Created a google drive folder: lc0 put in the lczero-training-master folder, Training Data folder, trainingdata-tool.exe & pgn-extract.exe and the input.pgn as described in https://lczero.org/dev/wiki/training-and-testing-a-net-on-google-colab-beginner-friendly-guide/

with google colab created a notebook & with this commands it finally worked:
from google.colab import drive
drive.mount('/content/gdrive', force_remount=True)
!chmod -R 777 "/content/gdrive/My Drive/lc0/"
!"/content/gdrive/My Drive/lc0/lczero-training-master/tf/train.py" --cfg "/content/gdrive/My Drive/lc0/lczero-training-master/tf/configs/example.yaml" --output "/content/gdrive/My Drive/lc0/lczero-training-master/tf/tmp/mymodel4.txt"

**Comment by CallOn84** [comment]
The links for the .exe have all been depreciated. Luckily, I downloaded a copy before the link died. It's available in my repository for all my Lc0 nets.
